### PR TITLE
[test][cli] adding some cli metric tests to increase coverage

### PR DESCRIPTION
**Pull request by ryandeivert**
to @austinbyers 
size: small

## Changes
* Adding until tests for cli metric toggling and cli metric alarm creation.

### CODE REVIEW

File: tests/unit/stream_alert_cli/test_config.py [hunk 1]
```diff
@@ -72,8 +36,149 @@ def test_load_config():
...
+    config_data = basic_streamalert_config()
+
+    global_file = 'conf/global.json'
+    global_contents = json.dumps(config_data['global'], indent=2)
+
+    lambda_file = 'conf/lambda.json'
+    lambda_contents = json.dumps(config_data['lambda'], indent=2)
+
+    with mock_open(global_file, global_contents):
+        with mock_open(lambda_file, lambda_contents):
```
Comment (austinbyers) [inline tests/unit/stream_alert_cli/test_config.py]: A Python3 feature which has also been backported to 2.7:

```python
with mock_open(global_file, global_contents), mock_open(lambda_file, lambda_contents):
   # do stuff
```

Comment (austinbyers) [reply tests/unit/stream_alert_cli/test_config.py]: Here and throughout (this is in every test you added)

File: tests/unit/stream_alert_cli/test_config.py [hunk 2]
```diff
@@ -72,8 +36,149 @@ def test_load_config():
...
+@patch('stream_alert_cli.config.CLIConfig.write')
+def test_toggle_metric(write_mock, log_mock):
+    """CLI - Metric toggling"""
+    config_data = basic_streamalert_config()
+
+    global_file = 'conf/global.json'
+    global_contents = json.dumps(config_data['global'], indent=2)
+
+    lambda_file = 'conf/lambda.json'
+    lambda_contents = json.dumps(config_data['lambda'], indent=2)
```
Comment (austinbyers) [inline tests/unit/stream_alert_cli/test_config.py]: Building these contents happens for every single test. Consider making a TestingClass and put this in the `setup` (to reduce code de-duplication)

File: tests/unit/stream_alert_cli/test_config.py (hunk 1 above)
Comment (ryandeivert) [reply tests/unit/stream_alert_cli/test_config.py]: Nice i hadn't realized that was backported. Was just copy pasta from some tests someone else added

**Comment by ryandeivert** [comment]
@austinbyers PTAL again - I updated tests to reduce duplication

**Pull request merged**